### ISSUE TITLE
Functions, depend on SampleImageData don't work 

### ISSUE THREAD

**Issue by KsanfFillum** [Blizzard/s2client-api#317]
There are some calls that depend on `SampleImageData` function, for example `Observation()->IsPlacable` or `Obervation()->HasCreep`. All of them are throwing an assert here:
```
sc2_client::SampleImageData(const std::string& data, int width, int height, const Point2D& point, unsigned char& result):

assert(data.size() == width * height); (Line 439)
```

There is no difference on which frame or function you call this.

**Comment by mboedigh** [comment]
I noticed it too. The issue seems to be that sc2 protocol is returning 1 bit images, and the c++ sc2 api can't cope
I put some sample code that fixed the problem for me in a previous issue #311. I don't know how to make a pull request (and have also changed other things so I don't think I should anymore), or I would have uploaded it.

**Comment by KsanfFillum** [comment]
> I noticed it too. The issue seems to be that sc2 protocol is returning 1 bit images, and the c++ sc2 api can't cope
> I put some sample code that fixed the problem for me in a previous issue #311. I don't know how to make a pull request (and have also changed other things so I don't think I should anymore), or I would have uploaded it.

Found it, thanks. For some reason i see that developers decide to to merge any pull request, but thank you for your solving. I think i close my issue.
